integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -854,7 +858,7 @@ void testHiveCatalogCreatedByGravitino() throws InterruptedException {
   }
 
   @Test
-  void testWrongHiveCatalogProperty() throws InterruptedException {
+  void testWrongHiveCatalogProperty() {
```

Comment:
@yuqi1129 , I wonder why `checkTrinoHasLoaded(sql, 6);`  is called in this test. It seems that it will never be executed successfully in this case.  Should we just remove this line or use `await().atLeast(6, TimeUnit.SECONDS)` instead?

Reply:
@xiaozcy 
The initial goal was to wait for 6 seconds to ensure that Trino had tried to load catalogs several times, so I think we can use `await().atLeast(6, TimeUnit.SECONDS)` instead directly.